content/articles/understanding-how-cluster-marker-works/index.md

```diff
@@ -0,0 +1,211 @@
+### Understanding how cluster markers work in web mapping using leaflet.js
+ Big data is the buzzword of the twenty-first century. Only a few people recognize that a major portion of the data is classified as geographical. Even the most basic scenario of showing data and markers on a map with a satisfactory user experience can be a major issue. The marker clustering method is recommended for improving this type of map.
```

Comment:
A statement like this needs a source... and I'm more than half way into your introduction and have no clue what the article is about. @Adebogunabdulroheem